lambda.tf

```diff
@@ -13,7 +13,7 @@ resource "aws_lambda_function" "publish_egress_ip" {
   handler          = "lambda_handler.handler"
   memory_size      = 128
   role             = aws_iam_role.lambdaexecution_role.arn
-  runtime          = "python3.9"
+  runtime          = "python3.12"
```

Comment:
Is there any reason not to use the Python 3.13 runtime?

Reply:
There is not.  Thanks for noticing that.  Updated the PR description and added https://github.com/cisagov/publish-egress-ip-terraform/pull/54/commits/7b39a9688f0ab1fa8f1c19318cf9a0047a160be2.